### PR TITLE
CI tests

### PR DESCRIPTION
**Pull request by enhaut**
Pushes and pull requests to `master` triggers Github action,
which tries to merge upstream `master` branch and pull request's
source branch and also runs `SimpleNetworkRecipe` using
2 containers whose try to ping eachother. `runner.py` is taken from
our [wiki](https://lnst.readthedocs.io/en/latest/installation.html#creating-an-executable-helloworld-test-script).

GitHub action [does not support Fedora](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), so I had to use Ubuntu instead,  
basically the requirements are the same but some of them have quite different name.
Actions have [limits](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits) but it's pretty generous so I don't think it would be problem in future.

### CODE REVIEW

**Comment by enhaut** [comment]
@olichtne at the end, I've removed rebasing to upstream master, github also checks the merging ability and it won't trigger the action if PR is not able to merge automatically so doing it again is not useful at all.  
I've used `action/checkout` to checkout to source branch, regards to it's [documentation](https://github.com/actions/checkout#usage) it checkouts to `HEAD` of branch that triggers the action.
  
[Here](https://github.com/enhaut/lnst/runs/5376985573?check_suite_focus=true) is log of activity triggered by pull request to my forked LNST repo.

**Review by olichtne — DISMISSED** [review]
Looks good to me.

Before merging I'll let others comment too @Axonis  @pgagne @jtluka

File: .github/workflows/ci-test.yml
Comment (pgagne) [inline .github/workflows/ci-test.yml:54]: Shouldn't this be `LNST agent image`

Comment (enhaut) [reply .github/workflows/ci-test.yml]: That was just typo, it should be `LNST agents`, thanks

**Comment by pgagne** [comment]
Ack looks good.

**Review by pgagne — APPROVED** [review]
looks good.

**Review by olichtne — APPROVED** [review]
reviewed by everyone, merging

**Pull request merged**
